source/faq.txt

```diff
@@ -49,13 +49,18 @@ clusters.
 Should I increase the size of the ``oplog`` in the source cluster?
 ------------------------------------------------------------------
 
-The :term:`oplog` is a capped collection that keeps a rolling 
-record of all operations that modify the data stored in your databases. 
-
-.. include:: /includes/fact-oplog-background
-
-To learn more about how to increase the size of the ``oplog``, see:
-:ref:`tutorial-change-oplog-size`. 
+You might exceed the oplog window and need to increase the size of the
```

Comment:
I think we should include the info about ORR (from the other section) here. This way, users are less alarmed and are aware that mongosync is resilient to this issue.

Reply:
Done!